src/clientAuth/bearerHeader.js

```diff
@@ -6,49 +12,37 @@ class BearerHeader extends ClientAuthnMethod {
   }
 
   /**
-   * More complicated logic then I would have liked it to be
-   * @param {*} cis Request class instance
-   * @param {*} ci Client information
-   * @param {*} requestArgs: request arguments
-   * @param {*} httpArgs HTTP header arguments
-   * @param {*} kwargs
+   * @param {?ResourceRequest} cis Request class instance
+   * @param {?ClientInfo} cliInfo Client information
+   * @param {?Object<string, string>} httpArgs HTTP header arguments
+   * @param {?Object<string, string>} kwargs Other optional arguments
```

Comment:
This lacks a @return parameter.

But side question: why returning the httpArgs in this method? You mutate it anyway, no need to returning it.

Reply:
Have to return it as I don't always pass in an httpArgs as a parameter
const bh = new BearerHeader();
const cis = new ResourceRequest({access_token: 'Sesame'});
const httpArgs = bh.construct(cis, client.clientInfo);
like here for example